tools/create_android_makefiles

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Run this script ONLY inside an Android build system
+# and after you ran lunch command!
+
+if [ -z "$ANDROID_BUILD_TOP" ]; then
+  echo "Run lunch before running this script!"
+  exit 1
+fi
+
+if [ -z "$1" ]; then
+  ARCH="arm"
+else
+  ARCH="$1"
+fi
+
+if [ $ARCH = "x86" ]; then
+  TARGET_ARCH="ia32"
+else
+  TARGET_ARCH="$ARCH"
+fi
+
+./configure \
```

Comment:
You need to update the paths now that you've moved the file.  Alternatively (and preferably), add a `cd $(dirname "$0/..")` at the top, that makes it independent of the current working directory.

Tiny style nit: can you indent the line continuations with four instead of two spaces?  Thanks.